rich/pretty.py

```diff
@@ -774,6 +805,7 @@ def pprint(
         max_length (int, optional): Maximum length of containers before abbreviating, or None for no abbreviation.
             Defaults to None.
         max_string (int, optional): Maximum length of strings before truncating, or None to disable. Defaults to None.
+        max_depth (int, optional): Maximum depth for nested data structures, or None for no maximum. Defaults to None.
```

Comment:
This sounds clearer to me
```suggestion
        max_depth (int, optional): Maximum depth for nested data structures, or None for unlimited depth. Defaults to None.
```